File: chapter6/exercise9/main.ml
type pair = int * int;
type point = int * int;
val x: point = (1, 4);
val y: pair = x;
type person = {name: string, age: int};
type school = {name: string, age: int};
val p: person = {name="Alice", age=22};
val s: school = p;

File: chapter6/exercise9/main.cpp
#include <stdio.h>

typedef struct {
   int a;
   int b;
} Point;

typedef struct {
   int a;
   int b;
} Pair;


int main () {
  // Name equivalence
  Point x;
  Pair y;
  // x = y; // error
  return 0;
}
